experimental/envoy_proxy/README.md

```diff
@@ -7,8 +7,14 @@ in Envoy proxy.
 
 The experiment consists of 2 Envoy proxies: _client side_ and _server side_.
```

Comment:
Perhaps you could add the fancy new diagram with the pipes here too :) Or in another PR if you prefer.